vcd/resource_vcd_lb_service_monitor_test.go

```diff
@@ -0,0 +1,147 @@
+// +build gateway lb lbsm ALL functional
```

Comment:
I thought I posted this, but looks like the comment got lost between the tabs. "lb" abbreviation is OK, but "lbsm" is too much for my taste :) Maybe let's expand into full "service monitor" wording?
CC: @dataclouder

Reply:
Agree, let's avoid to much acronyms

Reply:
It's the same as in govcd now. `lbServiceMonitor`